### DISCUSSION TITLE
Android 15 Support

### DISCUSSION THREAD

**Discussion by ghost** [LawnchairLauncher/lawnchair#4892] [General]
Will Lawnshair be supported in Android 15?

**Comment by validcube** [comment]
Lawnchair 14 supports Android 8.1 and newer, yes, which includes Android 15.

**Reply by ghost** [reply]
But it does not support all Android 15 features like private space.

**Reply by validcube** [reply]
> But it does not support all Android 15 features like private space.

Ah whoops, sorry, I thought you meant can you run Lawnchair 14 on Android 15.

At the moment, you can try out [Lawnchair 15 development build](https://github.com/LawnchairLauncher/lawnchair#development-builds), I can't try out Android 15 specific features yet on my Samsung device but I'm sure that support for Private Space might not be ready yet.

Support for Android 15 features _may be_ available once it's ready for release.

**Reply by ghost** [reply]
Ah ok, thanks for the info.
I use GrapheneOS on a Google Pixel Phone. GrapheneOS currently uses the standard AOSP Launcher.
For us GrapheneOS users, there are only few alternatives, but we are looking forward for the Lawnchair 15 release.
